app/Http/Controllers/Site/HomePageController.php

```diff
@@ -17,9 +17,13 @@ public function __invoke()
 
     private function nextEvent()
     {
-        $html = file_get_contents('https://cts.vatsim.uk/extras/next_event.php');
+        try {
```

Comment:
What sort of errors are you getting to result in the need for a try catch here?

Reply:
Mainly to do with SSL handshake timeout issues / dodgy connections. Have found that this caused phpunit to error on my tests a couple of times

Reply:
Would we be better returning something to avoid the homepage looking foobar if it does throw an exception?

Reply:
Returning null causes it to show no upcoming events which is fine

Reply:
So it does! I built it properly then ;)